website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -435,12 +431,19 @@ memcached-sample                        5/5     5            5           3m
 
 ### Cleanup
 
+A new target can be added into the Makefile for cleaning up the resources that have been created along this tutorial:
+
 ```sh
-$ kubectl delete -f config/samples/cache_v1alpha1_memcached.yaml
-$ kubectl delete deployments,service -l control-plane=controller-manager
-$ kubectl delete role,rolebinding --all
+# Undeploy controller from the configured Kubernetes cluster in ~/.kube/config
+undeploy:
+	$(KUSTOMIZE) build config/default | kubectl delete -f -
 ```
 
+Once that's done the simple command below will delete all the resources:
+
+```sh
+$ make undeploy
```

Comment:
```suggestion
make undeploy
```

Reply:
every single command in the tutorial is preceded with "$" as such I think that we should keep it here as well

Reply:
IMO, we can normalize this in a follow on. The preferred format is without the `$` for easy copy/paste, but if that's how everything else in the file looks right now, then I think it's outside the scope of this PR.

Reply:
Lets normalize in a follow-up.